src/main/kotlin/uk/gov/dwp/dataworks/exceptions/ErrorWhileProcessingRequestException.kt

```diff
@@ -0,0 +1,4 @@
+package uk.gov.dwp.dataworks.exceptions
+
+/** Used when specified request cannot be executed. */
+class ErrorWhileProcessingRequestException (request: String) : Exception("Error while processing the $request request")
```

Comment:
This is not specific enough to be an exception. Ideally they should be something like `FailedToCreateUserException` or `IllegalRequestParameterException`. Note that these are just examples